certificate/certificate.go

```diff
@@ -36,8 +37,8 @@ func getLocalhostCertAndKey(crtPath string, keyPath string) (cert []byte, key []
 }
 
 // UseLOCALHOST loads the certificate and private key as a TLS certificate.
-func UseLOCALHOST() (tlsCert *tls.Certificate, err error) {
-	cert, key, err := getLocalhostCertAndKey(os.Getenv("LOCALHOST_CERT"), os.Getenv("LOCALHOST_KEY"))
+func UseLOCALHOST(certPath string, keyPath string) (tlsCert *tls.Certificate, err error) {
+	cert, key, err := getLocalhostCertAndKey(certPath, keyPath)
```

Comment:
We should probably look at optimizing how .env is being used, so we maybe can support both using the makefile and not using the makefile. As long as we don't have to do too much for every test...